scripts/manifests/firefox.json

```diff
@@ -49,7 +56,7 @@
     }
   },
 
-  "content_security_policy": "default-src 'self'",
+  "content_security_policy": "default-src 'self'; style-src 'self' 'unsafe-inline'",
```

Comment:
I tested and this change is needed for Thunderbird as well.
![image](https://github.com/user-attachments/assets/d8a7dc98-098a-47c4-b1a4-0160fc6ca4a9)

Reply:
For Chrome it is not?

Reply:
I presume it would be required in Chrome as well, but I did not test it. Chrome dropped support for MV2, so I am not sure if it is possible to test anymore.